app/util/airportDetailFilter.js

```diff
@@ -0,0 +1,4 @@
+// pass in the airport code and the db connection (which includes the escape function)
+module.exports = (airportCode, db) => {
+    return db.escape(airportCode);
```

Comment:
Since this function `return`s, you can use an even more concise syntax: something like `(airportCode, db) => db.escape(airportCode)`. (But test it, just in case.)

Reply:
It works. I tried it.